values.yaml

```diff
@@ -0,0 +1,8 @@
+pachd:
```

Comment:
This is probably to commit, but let's name it different and maybe an an `example-values-files` repo (or something like that)